queries.sql

```diff
@@ -1,3 +1,25 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+select * from animals where RIGHT(nam
+e,3) = 'mon';
+
+select * from animals
+where date_of_birth > '2016-01-01' and date_of_birth < '2019-01-01';
+
+select name from animals
```

Comment:
- [ ] Kindly consider selecting neurtred = 'true' explecitly

Reply:
Could you pls explain why I need to do that explicitly?